servant-foreign/servant-foreign.cabal

```diff
@@ -27,10 +27,10 @@ source-repository head
 
 library
   exposed-modules:     Servant.Foreign, Servant.Foreign.Internal
-  build-depends:       base       == 4.*
-                     , lens       == 4.*
-                     , servant    == 0.5.*
-                     , text       >= 1.2  && < 1.3
+  build-depends:       base >=4.7 && <5
+                     , lens        == 4.*
+                     , servant     == 0.5.*
+                     , text        >= 1.2  && < 1.3
```

Comment:
Why was this changed? Both the whitespace and the narrower version bounds for base?